opennlp-tools/src/main/java/opennlp/tools/stemmer/snowball/arabicStemmer.java

```diff
@@ -30,2094 +30,1284 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 
  */
 
-// This file was generated automatically by the Snowball to Java compiler
-
+// Generated by Snowball (build from 867c4ec70debd4daa7fb4d5a9f7759b47887d0b9)
 package opennlp.tools.stemmer.snowball;
 
-
- /**
-  * This class was automatically generated by a Snowball to Java compiler
-  * It implements the stemming algorithm defined by a snowball script.
-  */
-
+/**
+ * This class implements the stemming algorithm defined by a snowball script.
+ * <p>
+ * Generated by Snowball (build from 867c4ec70debd4daa7fb4d5a9f7759b47887d0b9) - <a href="https://github.com/snowballstem/snowball">https://github.com/snowballstem/snowball</a>
+ * </p>
+ */
+@SuppressWarnings("unused")
 public class arabicStemmer extends AbstractSnowballStemmer {
```

Comment:
Seem strange per naming convention ! Shouldn't this be ArabicStemmer with cap A ?

Reply:
Ideally, we should make them `package-private` and rename the classes, but open to thoughts